docs-new/docs/graphql/core/scheduled-triggers/create-cron-trigger.mdx

```diff
@@ -0,0 +1,186 @@
+---
+description: Create a cron trigger with Hasura
+keywords:
+  - hasura
+  - cloud
+  - cron trigger
+  - scheduled trigger
+  - create
+sidebar_position: 1
+---
+
+import Thumbnail from "@site/src/components/Thumbnail/Thumbnail";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Creating a cron trigger
+
+## Introduction
+
+Cron triggers are used to reliably trigger HTTP endpoints to run custom business logic periodically based on a [cron schedule](https://en.wikipedia.org/wiki/Cron). For example, you can create a cron trigger to generate an end-of-day sales report every weekday at 10pm.
+
+To add a cron trigger, follow these steps:
+
+## Step 1: Navigate to Cron Triggers
+
+- Go to the `Events` tab in your Hasura console.
+- Click `Cron Triggers`.
+- Click `Create`.
+
+<Thumbnail
+  src="/img/graphql/core/event-triggers/create-cron.png"
+  alt="Adding a cron trigger"
+  width="1000px"
+/>
+
+## Step 2: Define the cron trigger
+
+Let's define the cron trigger.
+
+<Tabs className="api-tabs">
+<TabItem value="console" label="Console">
+
+In the form opened by the above step, fill out the following fields:
+
+- **Name**: Create a name for the cron trigger.
+- **Webhook**: Enter the HTTP endpoint that should be triggered.
+- **Cron schedule**: Enter a schedule for the cron. You can use the link next to the field to help [build a cron expression](https://crontab.guru/#*_*_*_*_*), or use the `Frequently used crons` dropdown as a shortcut. Cron events are created based on the UTC timezone.
+- **Payload**: The JSON payload which will be sent to the webhook.
+
+<Thumbnail
+  src="/img/graphql/core/event-triggers/define-cron-trigger.png"
+  alt="Defining a cron trigger"
+  width="550px"
+/>
+
+In this example, we're creating a cron trigger called `eod_reports`, to trigger the webhook <a href="https://mywebhook.com/eod">`https://mywebhook.com/eod`</a>. The cron schedule is set to `0 22 * * 1-5`, which means "At 22:00 on every day-of-week from Monday through Friday" (you can check this [here](https://crontab.guru/#0_22_*_*_1-5)). 
```

Comment:
this is `https://mywebhook.com/eod` only a inline code, no link required pls remove anchor tag

Reply:
done